build-tools/firebase-deploy

```diff
@@ -13,18 +13,21 @@ main() {
 
 	environ=$(get_environment "${1:-stage}") || exit $?
 	channel=$(get_channel "${2:-}")
+
 	rm -rf out/firebase
 	cp -r firebase out/
-	cp -r frontend out/firebase/public
+
 	if [[ "${channel}" == "live" ]]; then
-		hash_css_js out/firebase/public
-		update_refs "${environ}"
+		local domain
+		[[ "${environ}" == "prod" ]] && domain="evy.dev" || domain="evystage.dev"
```

Comment:
Ah, so this is just a little bit of extra safety saying:
it's either gotta be evystage.dev or prod&evy.dev
I like that.

Reply:
this was the same logic as before, just condensed. It used to live in `update_refs` in the shell script.